### ISSUE TITLE
0.6.1 needs to be published to rubygems.org

### ISSUE THREAD

**Issue by danhunter** [damir/okta-jwt#7]
Could you please publish the 0.6.1 tag to rubygems? 

We can include it in our Gemfile using the git source, but that's not compatible with dependabot (as per https://github.com/dependabot/dependabot-core/issues/4002)

**Comment by camilopayan** [comment]
Yep! This would be a huge help. [Currently Rubygems is only aware of 0.6.0 as the latest version.](https://rubygems.org/gems/okta-jwt)

**Comment by danhunter** [comment]
@damir any chance this could be resolved?
